packages/eslint-plugin/src/configs/recommended.ts

```diff
@@ -11,5 +11,6 @@ export const recommended = {
     '@compiled/no-keyframes-tagged-template-expression': 'error',
     '@compiled/no-styled-tagged-template-expression': 'error',
     '@compiled/no-suppress-xcss': 'error',
+    '@compiled/no-empty-styled-expression': 'warning',
```

Comment:
This should be an error as it is considered best practice, and if it causes too many violations in Jira then you can just override it back to warning in product.